search-app/src/cmr/search/services/query_execution/has_granules_or_cwic_results_feature.clj

```diff
@@ -0,0 +1,87 @@
+(ns cmr.search.services.query-execution.has-granules-or-cwic-results-feature
+  "This enables the :has-granules-or-cwic feature for collection search results. When it is enabled
+  collection search results will include a boolean flag indicating whether the collection has
+  any granules at all as indicated by provider holdings."
+  (:require
+   [cmr.common-app.services.search.elastic-search-index :as common-esi]
+   [cmr.common-app.services.search.group-query-conditions :as gc]
+   [cmr.common-app.services.search.parameters.converters.nested-field :as nf]
+   [cmr.common-app.services.search.query-execution :as query-execution]
+   [cmr.common-app.services.search.query-model :as qm]
+   [cmr.common.cache :as cache]
+   [cmr.common.cache.in-memory-cache :as mem-cache]
+   [cmr.common.config :as cfg :refer [defconfig]]
+   [cmr.common.jobs :refer [defjob]]
+   [cmr.search.data.elastic-search-index :as idx]))
+
+(def REFRESH_HAS_GRANULES_OR_CWIC_MAP_JOB_INTERVAL
+  "The frequency in seconds of the refresh-has-granules-or-cwic-map-job"
+  ;; default to 1 hour
+  3600)
+
+(def has-granule-cache-key
+  :has-granules-or-cwic-map)
+
+(defconfig cwic-tag
+  "has-granules-or-cwic should also return any collection with configured cwic-tag"
+  {:default "CWIC"})
```

Comment:
We should get the name of the currently used CWIC tag from Mark Reese and use that as the default. I'm guessing it is:
"edsc.portal.cwic.collections"
but we should confirm.